cypress/e2e/topics/createEditDeleteTopic.feature

```diff
@@ -4,11 +4,12 @@
 @group-C
 @pr
 @pre_check_test
+@prio_0_staging
 Feature: Topics - To create, edit and delete topics by the teacher.
 
-    As a teacher I want to create, edit and delete a new topic so that the student can see it
+    As a teacher, I want to create, edit and delete a new topic so that the student can see it
 
-    Scenario Outline: Teacher creates, edits and deletes a topic in the course, including pre-conditions
```

Comment:
same for here and in other feature files too -->  _, including pre & post conditions_  to be added.